package.json

```diff
@@ -28,6 +28,8 @@
     "classnames": "2.2.5",
     "config": "1.24.0",
     "csv-parse": "1.1.10",
+    "expect": "1.20.2",
+    "inist-ark": "2.0.0",
```

Comment:
I was planning to use it to validate the config but it wasn't possible. I kept it anyway as we will use it later to generate ARK uris.